<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationsWindow</class>
 <widget class="QMainWindow" name="OperationsWindow">
  <property name="windowTitle">
   <string>File operations</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QGridLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QListWidget" name="operationList">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="emptyInfo">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>There are no file operations</string>
      </property>
      <property name="alignment">
       <enum>Qt::AlignHCenter | Qt::AlignVCenter</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="windowMenu">
    <addaction name="clearAction"/>
   </widget>
   <addaction name="windowMenu"/>
  </widget>
  <action name="clearAction">
   <property name="text">
    <string>Clear finished</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
